pages/api/metrics/[project_id].js

```diff
@@ -5,7 +5,7 @@ const conn = new PSDB('main')
 
 export default async (req, res) => {
   const { project_id } = req.query
-  const [measures] = await conn.query('select * from metric where project_id = ?', project_id)
+  const [measures] = await conn.query('select * from metric where project_id = ? order by priority ASC, id ASC', project_id)
```

Comment:
The id is a fallback to keep the current order for projects that didn't define priorities